### ISSUE TITLE
fps doen't show in the upper left of the screen

### ISSUE THREAD

**Issue by iamyoyz** [bolrog/d2dx#71]
Greetings Bolrog this is just a small bug I'm not sure if you experience this but everytime I type /fps and press enter it doesn't show how much fps I have in the upper left of the screen .... 

I use 1.14d client , basemod for 1.14d and plugy  14.02
![bug3](https://user-images.githubusercontent.com/74028788/118476439-1602bc00-b740-11eb-906e-e3b4b14b8399.jpg)

![bug1](https://user-images.githubusercontent.com/74028788/118476422-0edbae00-b740-11eb-9bb3-eaf1b0afe53c.jpg)
![Bug2](https://user-images.githubusercontent.com/74028788/118476428-11d69e80-b740-11eb-9fb1-615b91f50153.jpg)

**Comment by MAXBURAOT** [comment]
lol there is no /fps command in d2dx, if you use nglide it will work , ive mention that already on my previous post

Using nGlide
![5](https://user-images.githubusercontent.com/33173384/118481799-74cb3400-b746-11eb-82b7-c10a0290f7df.png)

**Comment by bolrog** [comment]
Yeah I plan to add a frametime graph and fps counter eventually. I've decided to focus on bugs at the moment since there have been some lately, but after that hopefully I can get it done.

**Comment by iamyoyz** [comment]
thanks for all your replies unfortunately I nglide doesn't work for me even svens glider but don't worry about it @bolrog  just take your time no pressure

**Comment by qqkookie** [comment]
I also hope that "/fps" and "/framerate" command will be implemented someday. It is such a handy tool.
